apps/designer/app/canvas/canvas.tsx

```diff
@@ -56,6 +53,101 @@ import {
   config,
 } from "~/canvas/features/wrapper-component/text-editor";
 
+// TODO: move this somewhere else
```

Comment:
we use @todo for consistency
```suggestion
// @todo move this somewhere else
```

Reply:
here is a good location for it: app/canvas/shared/use-drag-drop.ts